src/platform/qpg6100/qpg6100Config.cpp

```diff
@@ -284,6 +255,33 @@ CHIP_ERROR QPG6100Config::ForEachRecord(Key firstKey, Key lastKey, bool addNewRe
     return err;
 }
 
+CHIP_ERROR QPG6100Config::MapNVMError(qvStatus_t aStatus)
+{
+    CHIP_ERROR err;
```

Comment:
Supernit: if you return instead of set+break, code is shorter:

```cpp
    switch (aStatus)
    {
    case QV_STATUS_NO_ERROR:
        return CHIP_NO_ERROR;
    case QV_STATUS_BUFFER_TOO_SMALL:
        return CHIP_ERROR_BUFFER_TOO_SMALL;
    case QV_STATUS_INVALID_ARGUMENT:
        return CHIP_ERROR_INVALID_ARGUMENT;
    case QV_STATUS_KEY_LEN_TOO_SMALL:
        return CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND;
    case QV_STATUS_INVALID_DATA:
        return CHIP_ERROR_PERSISTED_STORAGE_VALUE_NOT_FOUND;
    default:
        break;
    }
    return CHIP_ERROR_INTERNAL;
```